ptx/src/emit.rs

```diff
@@ -2098,6 +2099,21 @@ fn emit_inst_madcc(
         mul_result,
         arg.src3,
     )
+    /*
```

Comment:
Remove this comment. Feel free to also remove the comment in `emit_inst_madc`, I forgot to remove it earlier